src/Service.Tests/Authorization/AuthorizationResolverUnitTests.cs

```diff
@@ -1115,8 +1085,8 @@ public void ParsePolicyWithDuplicateUserClaims(bool exceptionExpected, params st
             AuthorizationResolver authZResolver = AuthorizationHelpers.InitAuthorizationResolver(runtimeConfig);
             Mock<HttpContext> context = new();
 
-            //Add identity object to the Mock context object.
-            ClaimsIdentity identity = new(TEST_AUTHENTICATION_TYPE, TEST_CLAIMTYPE_NAME, AuthenticationConfig.ROLE_CLAIM_TYPE);
+            //sAdd identity to the readAction, updateActionext.
```

Comment:
There are a few erroneous instances of this comment in this file that should be reverted or removed: `Add identity to the readAction, updateAction`